packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -261,10 +278,41 @@ const metadata: Meta<WaferMapArgs> = {
                 }
             }
         },
+        gridMinX: {
+            name: 'gridMinX',
```

Comment:
It looks like this story is inconsistent about referring to its API by the property syntax (`gridMinX`) vs the attribute syntax (`grid-min-x`). Lots of our other stories are inconsistent too but generally we prefer the attribute syntax when an API supports both.